lib/castle_web/views/api/podcast_view.ex

```diff
@@ -44,6 +44,10 @@ defmodule CastleWeb.API.PodcastView do
         href: api_podcast_download_path(conn, :index, podcast.id) <> "{?interval,from,to}",
         templated: true,
       },
+      "prx:listeners": %{
+        href: api_podcast_listener_path(conn, :index, podcast.id) <> "{?interval,from,to}",
```

Comment:
I see the route defined, https://github.com/PRX/castle.prx.org/blob/master/lib/castle_web/router.ex#L50

and the controller has tests - I think you're good to go.